common/onepanel/base/vars.yaml

```diff
@@ -21,9 +21,26 @@ database:
     required: true
     default: postgres
 application:
-  webHost:
+  host:
     required: true
     default: localhost
-  webPort:
-    required: true
-    default: 80
\ No newline at end of file
+  local:
+    webHTTPPort:
+      required: true
+      default: 4200
+    apiHTTPPort:
+      required: true
+      default: 8888
+    apiGRPCPort:
+      required: true
+      default: 8887
+  cloud:
+    uiPath:
+      required: true
+      default: web
```

Comment:
let's add more descriptive default values here and below.

Reply:
Here are the defaults for these:
uiPath: `/`
apiPath: `/api`
apiGRPCPort: `8887`

Reply:
I made this change in another branch

Reply:
change made in #22